xwiki-platform-core/xwiki-platform-security/xwiki-platform-security-requiredrights/xwiki-platform-security-requiredrights-ui/src/main/resources/js/security/requiredrights/requiredRightsDialog.js

```diff
@@ -241,6 +241,8 @@ define('xwiki-requiredrights-dialog', [
                 event.preventDefault();
                 this.save();
             });
+
+            this.#setupResponsiveLayoutInitialization();
```

Comment:
I guess this method is called to perform something equivalent to the removed CSS, and to avoid having hardcoded width in the media queries, but it's not so clear to me by reading the code.
Also wdyt of explaining the technical constraints forcing us to do something we usually do in CSS?

Reply:
There is a comment below `#setupResponsiveLayout` that might explain what we do here. See also my comments in the PR description, I'm not 100% sure it's the best way to do it.